scout/server/blueprints/institutes/views.py

```diff
@@ -150,11 +150,11 @@ def gene_variants(institute_id):
     if (form.hgnc_symbols.data) and len(form.hgnc_symbols.data) > 0:
         for hgnc_symbol in form.hgnc_symbols.data:
             if hgnc_symbol.isdigit():
-                hgnc_gene = store.hgnc_gene(int(hgnc_symbol))
-                if hgnc_gene is None:
+                hgnc_gene_caption = store.hgnc_gene_caption(int(hgnc_symbol))
```

Comment:
This way, don't you get only gene variants for genes in build 37? I've commented also above about this

Reply:
Yes! It was the same before, but should be fixed.

Reply:
Hm, ah, that is because it iss gene_variants, so variants could be from cases with either.